docling_serve/__main__.py

```diff
@@ -1,4 +1,4 @@
-import importlib
+import importlib.metadata
```

Comment:
im not sure how it works in docling_serve, it fails in REPL:
```python
>>> import importlib
>>> importlib.metadata.version
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'importlib' has no attribute 'metadata'
```
Also pyright was unhappy about previous import